Alert/doc/spec.md

```diff
@@ -0,0 +1,148 @@
+| ------------- | ------------- |------------- | ------------- | ------------- |------------- |
+| Standstill Traffic Jam |Minor Accident|Rainfall|Visible pollution (smog)|Symptoms or discomforts of users| People with asthma attacks at specific point. |
+| Heavy Traffic Jam |Car Accident|Temperature (low, high)|Level of emissions|Level of pollen on air| |
+| Bumper to bumper traffic|Hazard On Road|Heat alert|Level of pollutants, such as O3.|Pollen concentration| |
+| Moderate Traffic Jam|Assaults|Wind|Pre-contingency alert| | |
+| Car Stopped On Road|Bikers injured|Flood warning|Contingency alert| | |
+| Roadworks|Car crashes|dense fog|| | |
+| Road closed|Suspicious activities|Tornado | | | |
+| Pothole| |Tropical cyclone| | | |
+| Congestion on road| |Hurricane| | | |
+| Car in opposite way| |Tsunami| | | |
+| When a car suddenly stops | | | | | |
+
+In order to represent at data model the subtype of alert, the attribute "eventObserved" have been created. In this attribute, the information of subtype of alert should be added. It is necessary to define a catalogue of alerts and sub-types of alerts.
+
+## Data model
+
+A JSON Schema corresponding to this data model can be found [JSON Schema](../schema.json)
+
++ `id` : Entity's unique identifier.
+
++ `type` : Entity type. It must be equal to `Alert`.
+
++ `alertType` : Define the type of alert (Traffic jam, accidents, weather conditions, high level of pollutants)
+	+ Attribute type: text
+    + Normative References: [https://schema.org/Event](https://schema.org/Event)
+    + Mandatory
+
++ `eventObserved` : Describe the subtype of alerts (Alert: Traffic jam, Subtypes of alerts: Standstill Traffic Jam, Heavy Traffic Jam,  Car Stopped On Road, and so on).
+	 + Attribute type: text
+	 + Normative References: [https://schema.org/subEvent](https://schema.org/subEvent)
+	 + Mandatory
+
++ `location` : Location of alert represented by a GeoJSON geometry.
+    + Attribute type: `geo:json`.
+    + Normative References: [https://tools.ietf.org/html/rfc7946](https://tools.ietf.org/html/rfc7946)
+    + Mandatory
+
++ `address` : Civic address of `Alert`
+    + Normative References: [https://schema.org/address](https://schema.org/address)
+    + Mandatory if `location` is not present.
+
++ `dateTime` : The date and time of this alert in ISO8601 UTCformat. It can be represented by an specific time instant or by an ISO8601 interval.
+	+ Attribute type: DateTime or an ISO8601 interval represented as Text.
+	+ Mandatory
+
++ `description` : A description of alert (Traffic jam in Paseo de la Reforma. Emergency services at place).
+	 + Attribute type: text
+	 + Normative References: [https://schema.org/description](https://schema.org/description)
+	 + Optional	 
+
++ `refUser` : reference to the User that generated an alert. Within FIWARE, this reference could point to the end of the FIWARE Identity Manager, where FIWARE user profiles are stored.
```

Comment:
I would remove here any reference to the FIWARE's Idm. Data models should be kept independent of particular implementations

Reply:
roger